apps/web/pages/bookings/[status].tsx

```diff
@@ -153,7 +153,8 @@ export default function Bookings() {
   const [animationParentRef] = useAutoAnimate<HTMLDivElement>();
 
   return (
-    <ShellMain hideHeadingOnMobile heading={t("bookings")} subtitle={t("bookings_description")}>
+    <ShellMain withoutSeo hideHeadingOnMobile heading={t("bookings")} subtitle={t("bookings_description")}>
```

Comment:
It looks like ShellMain doesn't support  adding HeadSeo at all(which seems like a bug that creeped in later). Shouldn't we simply support HeadSeo there and avoid import HeadSeo everywhere ShellMain is already used

Reply:
It's pretty cool idea to support HeadSeo there and avoid import it everywhere.